### ISSUE TITLE
Browser inconsistency with pasteHTML()

### ISSUE THREAD

**Issue by vitosamson** [yabwe/medium-editor#866]
See this fiddle: https://jsfiddle.net/vitosamson/9cmkgzwb/2/

Focus on the editor element, then hit the paste button twice. In Firefox, it creates two `<p>` elements, but in Chrome the `<p>`s seem to get conjoined into one element.

Is this some weird browser quirk?

**Comment by nmielnik** [comment]
Yeah, this definitely seems like some strange browser inconsistency (Safari and Chrome merge the pasted `<p>`, Firefox keeps each `<p>` separate, I didn't check IE)

Just thinking about it briefly, it seems like we would want the behavior that Firefox has implemented (and not what Chrome and Safari are doing), as it seems a bit unexpected to me for the browser to merge together the elements that you're pasting with existing elements on the page. If you were pasting just plain text, not wrapped in an element tag, I could see merging it into the neighboring element though.
